### ISSUE TITLE
是否可以添加event，供调用

### ISSUE THREAD

**Issue by HarborZeng** [gitalk/gitalk#231]
比如onCommentSucceed, onCommentPreviewed, onGithubAccountLogined, onEditing, 等等，会很有用吧

**Comment by lijianfeigeek** [comment]
同样希望增加！

**Comment by Hrishikesh-K** [comment]
Yes, this is a much needed feature. We can use events like `onCommentsLoad`, etc. if it's implemented. Please consider this.
